.github/workflows/tests_without_binary.yml

```diff
@@ -34,19 +34,19 @@ jobs:
       - name: Get required Python packages
         run: |
           cd main
-          python -m pip install --upgrade pip
+          python -m pip install pip==23.0
           pip install -r requirements-dev.txt
 
       - name: Install lightning.qubit device
         run: |
           cd main
-          pip install -e .
+          pip install -e . -vv
         env:
           SKIP_COMPILATION: True
 
       - name: Run PennyLane-Lightning unit tests
         run: |
           cd main/
-          pytest tests/ $COVERAGE_FLAGS
+          python -m pytest tests/ $COVERAGE_FLAGS
```

Comment:
I think we should be consistent with `python -m`. I am in favour of using it here, and we should also update all pip statements to use it too.

Reply:
Done! Let me know if I missed something.

Reply:
Ah, perfect, thanks